### DISCUSSION TITLE
Application for Homey PRO 2023

### DISCUSSION THREAD

**Discussion by zdenoblazicek** [emsesp/EMS-ESP32#1659] [Ideas]
could you integrate with Homey PRO 2023?
I have https://bbqkees-electronics.nl/product/gateway-e32-v2-ethernet-wifi-edition-v2/
with buderus boiler and RC25 thermostat
I'm trying to write homeyscript for now, but it would be more ideal to have an application

**Comment by bbqkees** [comment]
Although it's a bit of manual work here is a starting point: https://github.com/angas/se.assermark.ems-esp
